packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -37,9 +40,31 @@ export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
       activeAppearance,
       image: imageProps,
       badge: badgeProps,
+      initials: _initials,
     },
     state: {
       ...state,
     },
   };
 };
+
+export const getInitials = (name: string): string => {
+  if (!name) return '';
+  const nonWordRegExp = /\W+(?<!\S-)/g;
+  const wordsInBracesRegExp = /\(\w+\)/g;
```

Comment:
Might also want to check for square brackets

Reply:
I guess this would be something to pass by PM/align with whatever algorithm we decide to go with

Reply:
I asked EJ. But just in case I added square and curly braces